src/plugins/counter/counter.c

```diff
@@ -18,10 +18,11 @@
 typedef int Counter;
 #define COUNTER_FMT "%d"
 
-static Counter elektraCountOpen = 0;
 
 int elektraCounterOpen (Plugin * handle, Key * errorKey ELEKTRA_UNUSED)
 {
+	static Counter elektraCountOpen;
```

Comment:
Is that really any safer?

Reply:
AFAIK only the initialization of static variables is thread-safe, not the actual usage of them. So if you want thread safety here we should either use atomics or a mutex.

Reply:
No, it is not. The plugin is only for debug purposes so it is somewhat okay. But I am also okay with removing the plugin so that we can claim that Elektra is fully thread safe.

Reply:
If we need the plugin for tests, I'd say we keep it, but excluded from being installed. If we don't actually need the plugin, let's just remove it.